guides/ansible_engine/3-variables/README.md

```diff
@@ -211,7 +210,6 @@ Your new, improved playbook is done! Let's take a second look to make sure every
       service:
         name: httpd
         state: restarted
-{% endraw %}
```

Comment:
Unfortunately, as of right now we need the `{% raw %}` tag to be able to export to github pages properly. Without the raw tag, the double curly brackets are not properly formatted.